package.json

```diff
@@ -2,11 +2,14 @@
   "private": true,
   "scripts": {
     "build": "vitruvius build --src src --dest lib --ignore-packages \"**/integration-tests/**\"",
-    "clean": "rimraf ./packages/*/lib",
+    "prebuild": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "yarn run clean --silent && yarn run build --silent",
+    "postinstall": "yarn build --silent",
     "precommit": "lint-staged",
-    "test": "jest",
+    "setup": "lerna clean --yes && lerna bootstrap",
+    "storybook": "lerna run --scope=react-percy-storybook-integration-tests storybook --stream",
+    "test": "jest --ci --no-cache --coverage",
```

Comment:
jest caches transpiled modules - so whenever you update babel or change transformation options there is a chance that you will get stale results. These can be hard to find problems, especially for people not versed in jest, as other test runners usually don't do this, hence the flag. Happy to remove it if it holds up this PR.